### ISSUE TITLE
Battery icon "[  ?   ]" question mark

### ISSUE THREAD

**Issue by mperson91** [exelban/stats#1095]
Menu bar battery indicator is showing a question mark "?" inside the icon, could be an interference with AlDente app, a battery charge limiter, which I also recommend as a feature for Stats; along with automated temperature based fan activation trigger setting similar to TG Pro.

![Screen Shot 2022-09-23 at 5 14 01 PM](https://user-images.githubusercontent.com/114262476/192057620-5b212cc9-9a62-4594-acf2-aab757bb6f1c.png)

The icon on the right is from app called Battery Indicator, I doubt it is interfering with the icon of Stats on the left side.

Thanks for looking!

macOS Monterey 12.6 (M1 Pro chip)

**Comment by exelban** [comment]
Hi. Thanks for the feedback. How long did you use Stats?
As I can see it could be possible if the Stats does not obtain the battery level yet. Please share the screenshot of the battery module popup view.

**Comment by mperson91** [comment]
I unplugged the power cord from the MacBook and it seems like it is displaying it properly now. I will note though, the plug icon is being displayed instead of the percentage, probably due to AlDente charge limiter app.

![Screen Shot 2022-09-23 at 6 07 31 PM](https://user-images.githubusercontent.com/114262476/192063077-e270d645-7b42-4664-83cb-54e93836bd5e.png)

I will check again next time I shut off and start the MacBook again, this bug could only have happened when it is first started or turned on, but the question mark stayed there for over half hour before I unplugged. I have been using Stats for a few months mostly for CPU/GPU/RAM percentage overview, just today I decided to check out the other features such as battery indicator, temperatures, and fan speeds. However I am not entirely sure if I will continue on using those features as the other mentioned apps provide more fan control, and a visual percentage inside the icon of the battery when plugged in. Would love to see all these features all in this app!

Thanks for the reply!
